Flexbox2/index2.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style2.css">
+    <title>Flexbox 2</title>
+</head>
+<body>
+    <div class="container">
+        <div class="parent3">
+            <div class="parent2">
+                <div class="parent1">
+                    <div class="box1">Vertical...</div>
+                    <div class="box2">...tiles</div>
+                </div>
+                <div class="box3">Wide tile</div>
+            </div>
+            <div class="box4">Middle tile</div>
+        </div>
+        <div class="box5">Tall tile</div>
+    </div>
```

Comment:
Pentru a scapa de acel `margin-left: -200px;` care este mai mult un "hack" ca div-ul albastru sa fie langa celelalte, ar trebuii schimbate intre ele Midle tile si Wide tile, astfel incat Midle tile sa fie impreuna cu celelalte 2 mici si nu cel Wide.

<img width="787" alt="Screenshot 2024-06-01 at 16 12 24" src="https://github.com/sandulachemalina/Wantsome/assets/33319632/17cbab7d-d049-442b-8bd1-581b22426f21">

```suggestion
    <div class="container">
        <div class="parent3">
            <div class="parent2">
                <div class="parent1">
                    <div class="box1">Vertical...</div>
                    <div class="box2">...tiles</div>
                </div>
                <div class="box4">Middle tile</div>
            </div>
            <div class="box3">Wide tile</div>
        </div>
        <div class="box5">Tall tile</div>
    </div>
```